tests/expressions_case/tests.py

```diff
@@ -312,6 +312,30 @@ def test_in_subquery(self):
             transform=attrgetter('integer', 'integer2')
         )
 
+    def test_exists(self):
+        instance = CaseTestModel.objects.annotate(
+            annotation=Case(
+                When(Exists(FKCaseTestModel.objects.filter(fk=OuterRef('pk'), integer=2)), then=Value("2")),
+                When(Exists(FKCaseTestModel.objects.filter(fk=OuterRef('pk'), integer=1)), then=Value("1")),
+                default=Value(""),
+                output_field=CharField(),
+            ),
+        ).get(integer=1)
+        self.assertEquals(instance.annotation, "1")
```

Comment:
This is already covered in `expressions` tests

https://github.com/django/django/blob/9624703a06187060c1a494e533f3e27fed946de3/tests/expressions/tests.py#L749-L760